src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
```

Comment:
Sensitive data like username and password should not be hard-coded in the source files. It's better to load them from environment variables or configuration files.